test/xskbench/um/main.c

```diff
@@ -43,6 +43,104 @@ ConsoleCtrlHandler(
     return TRUE;
 }
 
+XDP_STATUS
+CxPlatXskCreate(
+    _Out_ HANDLE *Socket
+    )
+{
+    return XdpApi->XskCreate(Socket);
+}
+
+XDP_STATUS
+CxPlatXdpCreateProgram(
+    _In_ UINT32 InterfaceIndex,
+    _In_ const XDP_HOOK_ID *HookId,
+    _In_ UINT32 QueueId,
+    _In_ XDP_CREATE_PROGRAM_FLAGS Flags,
+    _In_reads_(RuleCount) const XDP_RULE *Rules,
+    _In_ UINT32 RuleCount,
+    _Out_ HANDLE *Program
+    )
+{
+    return XdpApi->XdpCreateProgram(
```

Comment:
Same as elsewhere, please follow local style: the function goes on its own line if the line needs to be broken.

Each parameter does not need its own line.

Reply:
Where can I find the local style? I don't quite get what should be fixed

Reply:
Hey, yeah, it isn't documented in XDP - you just have to look around other files and try to make the style look similar. The style does mostly follow our internal netio style guide - I'll share internally.